builtin/lookup.go

```diff
@@ -1185,6 +1193,16 @@ fs diff(fs base)
 # @return an option to push the filesystem to a registry.
 fs dockerPush(string ref) binds (string digest)
 
+# Compress the image as a eStargz image before pushing.
+#
+# eStargz is a lazily pullable image format. It is compatible to OCI/Docker
+# images so it can be pushed to standard container registries and runnable on
+# eStargz-agnostic runtimes like Docker.
+# See: https://github.com/containerd/stargz-snapshotter
+#
+# @return an option to compress image as eStargz before pushing.
+option::dockerPush stargz()
```

Comment:
Does it make sense to call the option `stargz` if we are using the `estargz` format? Not really familiar with the terminology, so maybe people use `stargz` to refer to the whole class of formats, in which case this seems okay.

Reply:
It's like saying `bpf` instead of `ebpf`. The snapshotter itself is called `stargz-snapshotter` so I opted for `stargz`... nevertheless I don't mind either way.